src/test/java/com/liveramp/cascading_ext/combiner/lib/TestHyperLogLogPlusBackwardsCompatibility.java

```diff
@@ -0,0 +1,29 @@
+package com.liveramp.cascading_ext.combiner.lib;
+
+import com.clearspring.analytics.stream.cardinality.HyperLogLogPlus;
+import org.apache.commons.io.IOUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.*;
```

Comment:
nit: don't wildcard import if it's not necessary?

Reply:
fixed